modules/base/src/processors/volumesource.cpp

```diff
@@ -98,6 +98,11 @@ VolumeSource::VolumeSource(InviwoApplication* app, const std::string& file)
     addProperties(file_, reader_, reload_, information_, basis_, volumeSequence_);
     volumeSequence_.setVisible(false);
 
+    // ensure that the file name is always serialized by setting the default to ""
```

Comment:
I think that a better solution could be to add the option of specifying the default value in the FileProperty constructor. Otherwise:
```suggestion
    // The FileProperty default state is the one provided in the constructor (filePath). The default state is not serialized. So, to ensure that a custom file path is serialied we set the default state to ""
```

Reply:
This will affect all our properties. In this case, FileProperty is based on TemplateProperty<std::string>. Will use your suggestion.

Reply:
No, would work by making the change in the FileProperty constructor?
https://github.com/inviwo/inviwo/blob/master/src/core/properties/fileproperty.cpp#L50